### ISSUE TITLE
Windows PE?

### ISSUE THREAD

**Issue by bgagan911** [mbusb/multibootusb#235]
Can I use Windows PE boot images as well? 
tools like Yumi?

**Comment by mbusb** [comment]
It should. You can report an issue if it does not.

**Comment by necrose99** [comment]
yes it will , however if you want to just have windows 10 iso or a prefix your hosed... 
there isnt a just add the windows 10 install ISO  or /windows-10-install/   <<< Install files here insted... 
Add menu for Windows 10 Installer... 
http://windowsmatters.com/2016/11/10/gandalfs-win10pe-x64-redstone-build-14393-version-10-24-2016/   pe in main root like thus , NTFS compression , GPT yes rufus will deliver , however trying to add more with <img src="https://goo.gl/images/Hf6vae">  however with Windows iso or prefix one cannot specify 
a larger usb key 64-256 , one can also customize wim files ie more drivers updates etc..   
Gandal is like Windows 10 pe but with more tools , you can also add more tools to programs folder on usb. 
Unfortunately as well , having separate ntfs partitions or fat32 would be a plus...  as PE vs Windows Install will Deathmatch each other ... and it won't be pretty... 

so far adding Linux Iso's to UEFI seams to work ok , ist just adding windows versions ie a Gandalf-PE or the like...  they tend to fight.  [rufus](https://github.com/pbatard/rufus) works fairly well , just however not as tayloired to multi-boot ,  multiple windows on a stick.... either...
